clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/view/ChangeCaptureViewWriter.java

```diff
@@ -196,6 +188,8 @@ synchronized private void initializeVeniceWriter(int version) {
     if (veniceWriter != null) {
       return;
     }
+    // We don't want any linger time for this producer
+    props.put(KAFKA_LINGER_MS, 0);
```

Comment:
Can we also allow overriding arbitrary pubsub configs?

Reply:
This is added in the VeniceView logic (basically adding viewParameters to props).  But, I think I should maybe remove this config in case we get cold feet.